cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/CruiseControlEndPoint.java

```diff
@@ -33,7 +33,8 @@
   REVIEW_BOARD(CRUISE_CONTROL_MONITOR),
   ADMIN(CRUISE_CONTROL_ADMIN),
   REVIEW(CRUISE_CONTROL_ADMIN),
-  TOPIC_CONFIGURATION(KAFKA_ADMIN);
+  TOPIC_CONFIGURATION(KAFKA_ADMIN),
+  RIGHTSIZE(KAFKA_ADMIN);
```

Comment:
Do we want to submit the endpoint to upstream CC?

AFAIK, this is for demo purpose, shouldn't this be excluded?

@efeg what do you think?

Reply:
@lmr3796 I am fine with having this endpoint in the upstream. When used with a supporting provisioner implementation, it enables users to trigger manual resource provisioning requests (e.g. increase partition count, add new broker VMs) via CC, which can be useful to the open source community.